sklearn/kernel_ridge.py

```diff
@@ -6,11 +6,11 @@
 
 import numpy as np
 
+from .base import BaseEstimator, RegressorMixin
+from .metrics.pairwise import pairwise_kernels
+from .linear_model.ridge import _solve_cholesky_kernel
 from .utils import check_X_y
-from sklearn.base import BaseEstimator, RegressorMixin
-from sklearn.metrics.pairwise import pairwise_kernels
-from sklearn.linear_model.ridge import _solve_cholesky_kernel
-from sklearn.utils.validation import check_is_fitted
```

Comment:
test should always use absolute imports in scikit-learn. It might not be the same version that the tests belong to, but it will be the version that the user actually gets.